doc/related_projects.rst

```diff
@@ -150,6 +159,17 @@ and tasks.
 - `multiisotonic <https://github.com/alexfields/multiisotonic>`_ Isotonic
   regression on multidimensional features.
 
+- `libOPF <https://github.com/LibOPF/LibOPF>`_ Optimal path forest classifier
+
+- `pyensemble <https://github.com/dclambert/pyensemble>`_ An implementation of
```

Comment:
I suspect this is better fit in the Auto-ML section. But you would do well to check that.